.s2i/bin/assemble

```diff
@@ -14,14 +18,11 @@ wget https://getcomposer.org/download/1.9.0/composer.phar
 # Install deps
 mv composer.json{-dist,}
 php composer.phar install --no-dev
+php bin/install-jsdeps.sh
 
 # remove composer file
 rm composer.phar
```

Comment:
```suggestion
rm composer.phar
rm /tmp/src/bin/file
```
what do you think? since we probably do not wana have build dependencies in the running container